### ISSUE TITLE
Images broken at https://library-dev.bdrc.io/tradition/bo/ (for test)

### ISSUE THREAD

**Issue by berger-n** [buda-base/public-digital-library#917]

**Comment by berger-n** [comment]
@eroux it seems the ssl certificate for [legacy.tbrc.org](http://legacy.tbrc.org) has expired so images can't be loaded from an https source
(but I guess we'd better transfer the images from `https://legacy.tbrc.org/browser/images/` to `library-dev`)

**Comment by eroux** [comment]
yes, let's transfer the images, the legacy website will shut down at the end of the year

**Comment by berger-n** [comment]
fixed: https://library-dev.bdrc.io/tradition/bo/

![image](https://github.com/user-attachments/assets/1a3fa548-5db3-46ad-8fb7-8c07589f47ea)
